src/kvstore/NebulaStore.cpp

```diff
@@ -101,7 +101,14 @@ bool NebulaStore::init() {
                             enginePtr->removePart(partId);
                             continue;
                         } else {
-                             partIds.emplace_back(partId);
+                             auto it = std::find(partIds.begin(), partIds.end(), partId);
+                             if (it != partIds.end()) {
+                                LOG(INFO) << "Part " << partId
```

Comment:
When could this happen?

Reply:
I am not sure how it happens.  But i notice one part be opened twice on the same host in our user's logs.